lib/kuikka_website/web/templates/layout/sidebar.html.eex

```diff
@@ -5,7 +5,20 @@
     <img src="/images/logo.svg" />
   </div>
   <nav class="g--12">
-    <a href="<%= forum_path(@conn, :index) %>">Forums</a>
+    <%= if topics = forum_topics()  do %>
+      <div class="nav-collapsible">
```

Comment:
Im not sure it's great idea to show forum pages in side panel as there can be many forum topics which make finding proper one hard. 

Wiki currently uses side panel, but it will most likely change to use separate page list